src/test/java/service/GameTextQuestServiceTest.java

```diff
@@ -0,0 +1,49 @@
+package service;
+
+import model.GameTextQuestResult;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class GameTextQuestServiceTest {
+
+    @Test
+    public void testProcessPlayerActionWithNullAction() {
+        GameTextQuestService gameService = new GameTextQuestService();
```

Comment:
You create this object in every test. Do you think you can find a way how to create it only once and use it for each test?